cdc_stream/cdc_fuse_manager.cc

```diff
@@ -95,7 +95,7 @@ absl::Status CdcFuseManager::Start(const std::string& mount_dir,
   if (!status.ok()) {
     return absl::NotFoundError(absl::StrFormat(
         "Required gamelet component not found. Make sure the files %s and %s "
-        "reside in the same folder as stadia_assets_stream_manager_v3.exe.",
+        "reside in the same folder as cdc_stream.exe.",
```

Comment:
Maybe that should become a constant, same as `kFuseFilename` and `kLibFuseFilename`?

Reply:
Done.